source/_patterns/01-molecules/15-loan-sections/15-mobile-already-in-basket.mustache

```diff
@@ -1,14 +1,7 @@
 <div class="mobile-lend-action show-for-medium-down row">
-	 <div class="basket-buttons mobile-already-in-basket" {{^amountInBasket}}style="display: none;"{{/amountInBasket}}>
+	 <div class="basket-buttons mobile-already-in-basket" {{^amountInBasket}}style="display: none !important;"{{/amountInBasket}}>
 		<div id="checkout-button" class="columns small-9">
-			<button kvtrackevent="Lending|Amount in basket - Proceed to checkout|Mobile Borrower Profile|{{business_id}}|true" class="button">
```

Comment:
Is it ok to lose this track event?

Reply:
Thanks for the review @jedlin-kiva. I have already added the new icons to the atoms-images-icons file. It was a miss on my part that I lost this tracking event though, thanks for catching that. I just added it back, so we're good now.